diff --git a/centos8/Dockerfile b/centos8/Dockerfile
index ed3c558..4149502 100644
--- a/centos8/Dockerfile
+++ b/centos8/Dockerfile
@@ -1,6 +1,9 @@
 # The centos8 builder
 FROM centos:8
 
+ENV http_proxy=http://proxy:8888
+ENV https_proxy=http://proxy:8888
+
 # check for updates and ugrade packages
 RUN dnf check-update || { rc=$?; [ "$rc" -eq 100 ] && exit 0; exit "$rc"; }
 RUN dnf -y update
@@ -15,7 +18,7 @@ RUN dnf -y install git wget tar make cmake gcc gcc-c++ pkgconfig perl-devel
 
 # Packages required for building EPICS
 RUN dnf -y install libtirpc-devel
-RUN dnf -y --enablerepo=PowerTools install re2c libusb-devel rpcgen
+RUN dnf -y --enablerepo=powertools install re2c libusb-devel rpcgen
 RUN dnf -y install libxml2-devel pcre-devel libXext-devel
 RUN dnf -y install libjpeg-devel readline-devel libusbx-devel
 
diff --git a/centos8/run_build.sh b/centos8/run_build.sh
index b6c3401..f2f422b 100755
--- a/centos8/run_build.sh
+++ b/centos8/run_build.sh
@@ -17,4 +17,10 @@ echo "Checking out config version $ADCORE_VERSION"
 git checkout -q $ADCORE_VERSION
 cd ..
 fi
+
+if [ "$USE_LOCAL_INSTALL_CONFIG" == "FALSE" ];
+then
 python3 -u installCLI.py -y -s -c Install-Configurations/$CENTOS8_CONFIG -i /epics -p
+else
+python3 -u installCLI.py -y -s -m -c /ansible_epics -p
+fi
diff --git a/config.env b/config.env
index 935e27a..5e64377 100644
--- a/config.env
+++ b/config.env
@@ -1,6 +1,6 @@
 # Customize URLs to installSynApps tool and Install-Configurations
 INSTALL_SYNAPPS_URL=https://github.com/NSLS-II/installSynApps
-INSTALL_SYNAPPS_BRANCH=master
+INSTALL_SYNAPPS_BRANCH=feature-lean-bundles
 INSTALL_CONFIG_URL=https://github.com/jwlodek/Install-Configurations
 INSTALL_CONFIG_BRANCH=master
 
@@ -13,9 +13,14 @@ UBUNTU18_CONFIG=configureDeb8
 UBUNTU19_CONFIG=configureDeb8
 UBUNTU20_CONFIG=configureDeb8
 CENTOS7_CONFIG=configureDeb9
-CENTOS8_CONFIG=configureCentOS8
+#CENTOS8_CONFIG=configureCentOS8
+CENTOS8_CONFIG=../../ansible_epics
 RHEL8_CONFIG=misc/just-base
 
 # NEWEST if you wish to auto-update to newest releases of 
 # modules, otherwise specify an ADCore version R3-8 or higher
 ADCORE_VERSION=R3-8
+
+# Set to TRUE if a custom INSTALL_CONFIG will be used from a
+# Docker volume. Set to FALSE to use the GitHub repository.
+USE_LOCAL_INSTALL_CONFIG=TRUE
diff --git a/run_container.sh b/run_container.sh
index 0879094..f227154 100755
--- a/run_container.sh
+++ b/run_container.sh
@@ -7,7 +7,7 @@ function run_container () {
     CONTAINER_NAME=$1
     echo "Starting $CONTAINER_NAME build..."
     
-    docker run --env-file ./config.env --cidfile "$CONTAINER_NAME"_id.cid isa/"$CONTAINER_NAME" 
+    docker run -v /tmp/ansible_epics_vol:/ansible_epics --env-file ./config.env --cidfile "$CONTAINER_NAME"_id.cid isa/"$CONTAINER_NAME"
 
     echo "Copying package..."
     CONTAINER_ID=$(cat "$CONTAINER_NAME"_id.cid)
